js/app.js

```diff
@@ -0,0 +1,63 @@
+
+// array of objects to store search and class data for loop to look through and take correct course of action
+var userEntry,
+    city = [
+  {
+    name: 'New York',
+    altName0: 'New York City',
+    altName1: 'NYC',
+    class: 'nyc'
+  },
+  {
+    name: 'San Francisco',
+    altName0: 'SF',
+    altName1: 'Bay Area',
+    class: 'sf'
+  },
+  {
+    name: 'Los Angeles',
+    altName0: 'LA',
+    altName1: 'LAX',
+    class: 'la'
+  },
+  {
+    name: 'Austin',
+    altName0: 'ATX',
+    altName1: 'austin',
+    class: 'austin'
+  },
+  {
+    name: 'Sydney',
+    altName0: 'SYD',
+    altName1: 'syd',
+    class: 'sydney'
+  },
+
+];
+
+$(function() {
+
+
+  $('#submit-btn').click(changeBackground);
+
+  function changeBackground(event) {
+    event.preventDefault();
+    // Gathers User entry
+    userEntry = $('#city-type').val().toLowerCase();
+    console.log(userEntry);
+    // loops through data in city obeject array and checks if
+    // user entry matches any one of the items in array
+    // if so change class to corresponding class
+    for (var i = 0; i < city.length; i++) {
```

Comment:
@chrislindor Great job in using a loop to dry up your code. Really good stuff here.